src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -97,12 +97,22 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
   }
 
   // get path to integration definition JSON files
-  const WSTRING integrations_paths =
+  WSTRING integrations_paths =
       GetEnvironmentValue(environment::integrations_path);
 
   if (integrations_paths.empty()) {
-    Warn("Profiler disabled: ", environment::integrations_path,
-         " environment variable not set.");
+    auto profiler_home_path =
+        GetEnvironmentValue(environment::profiler_home_path);
+
+    if (!profiler_home_path.empty()) {
```

Comment:
Simple fallback in the profiler: if `DD_INTEGRATIONS` was not set (as a result, it's the empty string), try `DD_DOTNET_TRACER_HOME` and if it is not empty, create an `integrations.json` path.

Reply:
We duplicate this logic in `integration_loader.cpp` just so we can raise an immediate error immediately here if the environment variables don't look right. Also, for testing purposes, it's much easier to do all of the environment variable processing again in `LoadIntegrationsFromEnvironment` so we can unit test it.